src/feedback-rating/feedback-rating.styles.tsx

```diff
@@ -0,0 +1,49 @@
+import { MediaQuery } from "src/media";
+import { Button } from "src/button";
+import { Color } from "src/color";
+import styled from "styled-components";
+
```

Comment:
requesting some amendments to styling so that the elements respect the parent width and can grow/shrink appropriately

tested on 320px wide and 540px wide viewports

![Screenshot 2022-12-20 at 9 23 04 AM](https://user-images.githubusercontent.com/16853062/208559790-e18d31aa-2ac6-40b8-a4c4-41f11c8b9ed2.png)


![Screenshot 2022-12-20 at 9 22 35 AM](https://user-images.githubusercontent.com/16853062/208559795-0e818261-dd9f-4a32-b046-4793f8a5fbda.png)

Reply:
Updated the submit button to shrink based on width. 
Updated the flex row to column should the screen be smaller than 38rem(the width of the component)

Reply:
noticed ChildContainer and SubmitButton have hardcoded widths on smaller screens, can those be simplified to a responsive width? e.g. `width: 21rem` can actually be `width: 100%` (using calc to adjust for margin if needed)

Reply:
updated to use responsive widths

Reply:
updated